XamlControlsGallery/ControlPages/CommandBarPage.xaml.cs

```diff
@@ -101,6 +128,7 @@ private void RemoveSecondaryCommands()
         {
             while (PrimaryCommandBar.SecondaryCommands.Count > 1)
             {
+                MultipleButtons = false;
```

Comment:
You don't need to set this inside the while loop. Not a big deal, but it would be slightly more efficient if this line executed only once :)

Reply:
Fixed that in the newest commit, in addition to being slightly more efficient the code (at least in my opinion) gets a bit more understandable. Thanks for bringing that up !

Reply:
This piece needs to be moved outside too, right?

Reply:
Oh yes totally missed that, I'll fix this right now.

Reply:
Now everything should be fine.